socrates_api/tasks.py

```diff
@@ -396,6 +396,9 @@ def extract_asset_from_raw(service_tag, final_step=False):
             enclosure_asset.is_valid(raise_exception=True)
             enclosure_asset.save()
 
+    if final_step:
+        post_extraction_tasks(service_tag)
```

Comment:
This should probably be `post_extraction_tasks.apply_async((service_tag,))` to create its own task if the playbook fails.